# HG changeset patch
# User John Donoghue <john.donoghue@ieee.org>
# Date 1443435877 14400
#      Mon Sep 28 06:24:37 2015 -0400
# Branch stable
# Node ID bbe6a846d8ac938b17764dc5175407c2e7714268
# Parent  7d7c91ddc736bec491f4e760cb12c00219e480d0
Include stdint.h in mex.h file (Bug #46062)

* libinterp/corefcn/mex.h: add #include <stdint.h>

diff --git a/libinterp/corefcn/mex.h b/libinterp/corefcn/mex.h
--- a/libinterp/corefcn/mex.h
+++ b/libinterp/corefcn/mex.h
@@ -59,16 +59,18 @@ typedef int bool;
 /* -V4 stuff */
 #if defined (V4)
 #define Matrix mxArray
 #define REAL mxREAL
 #endif
 
 #define mxMAXNAME 64
 
+#include <stdint.h>
+
 #include "mexproto.h"
 
 #if defined (__cplusplus)
 extern "C" {
 #endif
 
 #if defined (V4)
 void mexFunction (int nlhs, mxArray* plhs[], int nrhs, mxArray *prhs[]);
